### DISCUSSION TITLE
Fooocus + Colab + New Models (download)

### DISCUSSION THREAD

**Discussion by quasar136a8** [lllyasviel/Fooocus#2329] [General]
Hi, help me, explain how to upload realistic (basic as on PC) models to collab to work faster and with better quality (realistic)?

**Comment by mashb1t** [comment]
You can add `--preset realistic` after `--share` in the playbook and Fooocus will automatically download the realistic model for you. Happy prompting!

**Reply by quasar136a8** [reply]
<img width="436" alt="Screenshot 2024-02-23 025942" src="https://github.com/lllyasviel/Fooocus/assets/111929719/a218852d-95ab-4c7b-8f81-44c6319ef43b">
like this?

**Reply by poor7** [reply]
Yes.
